app/_layouts/base.njk

```diff
@@ -43,6 +43,9 @@
         }, {
           text: "Wiki",
           href: "https://dfedigital.atlassian.net/wiki/spaces/BaT/overview"
+        }, {
+          text: "Glossary",
+          href: "/glossary-of-terms"
```

Comment:
This link is wrong. Should be `/glossary`

Reply:
Updated 😬